t/plugin/plugin.t

```diff
@@ -39,17 +39,18 @@ __DATA__
 --- config
     location /t {
         content_by_lua_block {
-            local lfs = require("lfs")
-            for file_name in lfs.dir(ngx.config.prefix() .. "/../../apisix/plugins/") do
-                if string.match(file_name, ".lua$") then
-                    local expected = file_name:sub(1, #file_name - 4)
+            --local lfs = require("lfs")
```

Comment:
For debug?

Reply:
Yes, forget to recovery, i have checked that there is no same problem